src/pages/HabitTracker/Habits.tsx

```diff
@@ -0,0 +1,238 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import { useState, useEffect } from "react";
+import { Modal, Button, Form } from "react-bootstrap";
+import styled from "styled-components";
+import _ from "lodash";
+import { PageBackground } from "../../components/styled/PageBackground";
+import { Sidebar } from "../../components/Sidebar";
+import { links } from "../../common/SidebarLinks";
+import { HabitTile } from "../../components/habittracker/HabitTile";
+import { AddHabitTile } from "../../components/habittracker/AddHabitTile";
+import { StyledModal } from "../../components/styled/Modal";
+
+// Styled component for the habit list container
+const HabitListContainer = styled.div`
+  width: calc(100% - 190px);
+  height: calc(100vh - 76px);
+  overflow-y: auto;
+  overflow-x: hidden;
+  position: absolute;
+  top: 56px;
+  left: 190px;
+
+  @media (max-width: 768px) {
+    width: calc(100% - 70px);
+    left: 70px;
+  }
+`;
+
+const HabitList = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  padding: 20px;
+  margin-left: 10px;
+  width: 100%;
+  box-sizing: border-box;
+  overflow-x: hidden;
+`;
+
+// Types
+interface Habit {
+  name: string;
+  count: number;
+}
+
+type EditHabit = {
```

Comment:
`Habit` and `EditHabit` have the same interface. Can use the `Habit` interface everywhere.